src/mol-gl/webgl/texture.ts

```diff
@@ -407,8 +407,8 @@ export function loadImageTexture(src: string, cell: ValueCell<Texture>, texture:
 
 //
 
-export function createNullTexture(gl: GLRenderingContext, kind: TextureKind): Texture {
-    const target = getTarget(gl, kind);
+export function createNullTexture(gl?: GLRenderingContext): Texture {
+    const target = 3553;
```

Comment:
Are you sure ``gl.TEXTURE_2D`` will be the same across all platforms? Maybe replace this with ``gl?.TEXTURE_2D ?? 3553`` ?

Reply:
They are guaranteed to be the same value but this way it is less of a magic number :)